<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MicrobiomeBenchmarkData â€¢ MicrobiomeBenchmarkData</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="MicrobiomeBenchmarkData">
<meta property="og:description" content="MicrobiomeBenchmarkData">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MicrobiomeBenchmarkData</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.99.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/MicrobiomeBenchmarkData.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/datasets.html">Datasets in MicrobiomeBenchmarkData</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="MicrobiomeBenchmarkData_files/header-attrs-2.11/header-attrs.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MicrobiomeBenchmarkData</h1>
                        <h4 class="author">Samuel D. Gamboa-Tuz</h4>
                  <a class="author_email" href="mailto:#"></a><a href="mailto:Samuel.Gamboa.Tuz@gmail.com" class="email">Samuel.Gamboa.Tuz@gmail.com</a>
      
                              <h4 class="author">Levi Waldron</h4>
                        
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/blob/master/vignettes/MicrobiomeBenchmarkData.Rmd"><code>vignettes/MicrobiomeBenchmarkData.Rmd</code></a></small>
      <div class="hidden name"><code>MicrobiomeBenchmarkData.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      <p>Document summary</p>
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>The <code>MicrobiomeBenchamrkData</code> package provides a collection of datasets suitable for benchmarking of differential abundance methods in R.</p>
<p>The datasets are also available through <a href="https://sandbox.zenodo.org/record/946131">Zenodo (sandbox)</a> so they can be downloaded and used out of the R environment if preferred/needed.</p>
<p>The main characteristic of this collection of datasets is that they contain some biological truth that has been proved experimentally in the literature.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p>MicrobiomeBenchmarkData should be available through the BioConductor project.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># if (!"BiocManager" %in% installed.packages()[,"Package"])</span>
<span class="co"># install.packages("BiocManager")</span>

<span class="co">## Release version</span>
<span class="co"># BiocManager::install("MicrobiomeBenchmarkData")</span>

<span class="co">## Development version</span>
<span class="co"># BiocManager::install("waldronlab/MicrobiomeBenchmarkData")</span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData">MicrobiomeBenchmarkData</a></span><span class="op">)</span></code></pre></div>
</div>
<div id="accessing-datasets" class="section level1">
<h1 class="hasAnchor">
<a href="#accessing-datasets" class="anchor"></a>Accessing datasets</h1>
<p>Currently, there are 6 datasets available through the MicrobiomeBenchmarkData. These datasets are accessed through the <code>getDataset</code> function.</p>
<div id="print-avaialable-datasets" class="section level2">
<h2 class="hasAnchor">
<a href="#print-avaialable-datasets" class="anchor"></a>Print avaialable datasets</h2>
<p>If no arguments are provided, the list of available datasets is printed on screen and returned invisiby as a character vector:</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">dats</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getDataset.html">getDataset</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; 1 Beghini_2019_16S_smoking</span>
<span class="co">#&gt; 2 HMP_2012_16S_gingival_V13</span>
<span class="co">#&gt; 3 HMP_2012_16S_gingival_V35</span>
<span class="co">#&gt; 4 HMP_2012_16S_gingival_V35_subset</span>
<span class="co">#&gt; 5 HMP_2012_WMS_gingival</span>
<span class="co">#&gt; 6 Stammler_2016_16S_spikein</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Use vignette('datasets', package = 'MicrobiomeBenchmarkData') for a detailed description of the datasets.</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Use getDataset(dryrun = FALSE) to import all of the datasets.</span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">dats</span>
<span class="co">#&gt; [1] "Beghini_2019_16S_smoking"         "HMP_2012_16S_gingival_V13"       </span>
<span class="co">#&gt; [3] "HMP_2012_16S_gingival_V35"        "HMP_2012_16S_gingival_V35_subset"</span>
<span class="co">#&gt; [5] "HMP_2012_WMS_gingival"            "Stammler_2016_16S_spikein"</span></code></pre></div>
</div>
<div id="access-a-single-dataset" class="section level2">
<h2 class="hasAnchor">
<a href="#access-a-single-dataset" class="anchor"></a>Access a single dataset</h2>
<p>In order to import a dataset, the <code>getDataset</code> function must be used with the name of the dataset as the first argument (<code>x</code>) and the <code>dryrun</code> argument set to <code>FALSE</code>. The output is a list vector with the dataset imported as a TreeSummarizedExperiment object.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">smoking</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getDataset.html">getDataset</a></span><span class="op">(</span><span class="va">dats</span><span class="op">[</span><span class="fl">1</span><span class="op">]</span>, dryrun <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span>
<span class="co">#&gt; Warning: 1 row(s) couldn't be matched to the tree and are/is removed.</span>
<span class="co">#&gt; Finished Beghini_2019_16S_smoking.</span>
<span class="va">smoking</span>
<span class="co">#&gt; class: TreeSummarizedExperiment </span>
<span class="co">#&gt; dim: 1297 135 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(1297): New.ReferenceOTU90 New.ReferenceOTU78 ...</span>
<span class="co">#&gt;   New.CleanUp.ReferenceOTU11636 New.CleanUp.ReferenceOTU11642</span>
<span class="co">#&gt; rowData names(7): Domain Phylum ... Genus Species</span>
<span class="co">#&gt; colnames(135): NYDH0394 NYDH0314 ... NYDH0218 NYDH0064</span>
<span class="co">#&gt; colData names(186): dataset gender ... ohq_5_3cat dbq_10_3cat</span>
<span class="co">#&gt; reducedDimNames(0):</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span>
<span class="co">#&gt; rowLinks: a LinkDataFrame (1297 rows)</span>
<span class="co">#&gt; rowTree: 1 phylo tree(s) (1298 leaves)</span>
<span class="co">#&gt; colLinks: NULL</span>
<span class="co">#&gt; colTree: NULL</span></code></pre></div>
</div>
<div id="access-a-few-datasets" class="section level2">
<h2 class="hasAnchor">
<a href="#access-a-few-datasets" class="anchor"></a>Access a few datasets</h2>
<p>Several datasets can be imported simultaneously by giving the names of the different datasets in a character vector:</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">gingival</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getDataset.html">getDataset</a></span><span class="op">(</span><span class="va">dats</span><span class="op">[</span><span class="fl">2</span><span class="op">:</span><span class="fl">5</span><span class="op">]</span>, dryrun <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V13.</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V35.</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V35_subset.</span>
<span class="co">#&gt; Finished HMP_2012_WMS_gingival.</span>
<span class="fu"><a href="https://rdrr.io/r/utils/str.html">str</a></span><span class="op">(</span><span class="va">gingival</span>, max.level <span class="op">=</span> <span class="fl">1</span><span class="op">)</span>
<span class="co">#&gt; List of 4</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V13       :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V35       :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V35_subset:Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_WMS_gingival           :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span></code></pre></div>
</div>
<div id="access-all-of-the-datasets" class="section level2">
<h2 class="hasAnchor">
<a href="#access-all-of-the-datasets" class="anchor"></a>Access all of the datasets</h2>
<p>If all of the datasets must to be imported, this can be done by providing the <code>dryrun = FALSE</code> argument alone.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">mbd</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getDataset.html">getDataset</a></span><span class="op">(</span>dryrun <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span>
<span class="co">#&gt; Warning: 1 row(s) couldn't be matched to the tree and are/is removed.</span>
<span class="co">#&gt; Finished Beghini_2019_16S_smoking.</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V13.</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V35.</span>
<span class="co">#&gt; Finished HMP_2012_16S_gingival_V35_subset.</span>
<span class="co">#&gt; Finished HMP_2012_WMS_gingival.</span>
<span class="co">#&gt; Warning: No taxonomy_tree available for Stammler_2016_16S_spikein.</span>
<span class="co">#&gt; Finished Stammler_2016_16S_spikein.</span>
<span class="fu"><a href="https://rdrr.io/r/utils/str.html">str</a></span><span class="op">(</span><span class="va">mbd</span>, max.level <span class="op">=</span> <span class="fl">1</span><span class="op">)</span>
<span class="co">#&gt; List of 6</span>
<span class="co">#&gt;  $ Beghini_2019_16S_smoking        :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V13       :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V35       :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_16S_gingival_V35_subset:Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ HMP_2012_WMS_gingival           :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span>
<span class="co">#&gt;  $ Stammler_2016_16S_spikein       :Formal class 'TreeSummarizedExperiment' [package "TreeSummarizedExperiment"] with 14 slots</span></code></pre></div>
</div>
</div>
<div id="annotations-for-each-taxa-are-included-in-rowdata" class="section level1">
<h1 class="hasAnchor">
<a href="#annotations-for-each-taxa-are-included-in-rowdata" class="anchor"></a>Annotations for each taxa are included in rowData</h1>
<p>The biological annotations of each taxa are provided as a column in the <code>rowData</code> slot of the TreeSummarizedExperiment.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co">## In the case, the column is named as BIOSIS</span>
<span class="va">tse</span> <span class="op">&lt;-</span> <span class="va">mbd</span><span class="op">$</span><span class="va">HMP_2012_16S_gingival_V35_subset</span>
<span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html">rowData</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span>
<span class="co">#&gt; DataFrame with 892 rows and 7 columns</span>
<span class="co">#&gt;              SUPERKINGDOM      PHYLUM       CLASS           ORDER</span>
<span class="co">#&gt;               &lt;character&gt; &lt;character&gt; &lt;character&gt;     &lt;character&gt;</span>
<span class="co">#&gt; OTU_97.31247     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.44487     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.34979     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.34572     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.42259     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; ...                   ...         ...         ...             ...</span>
<span class="co">#&gt; OTU_97.44294     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.45429     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.44375     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.45365     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt; OTU_97.45307     Bacteria  Firmicutes     Bacilli Lactobacillales</span>
<span class="co">#&gt;                        FAMILY         GENUS      BIOSIS</span>
<span class="co">#&gt;                   &lt;character&gt;   &lt;character&gt; &lt;character&gt;</span>
<span class="co">#&gt; OTU_97.31247 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.44487 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.34979 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.34572 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.42259 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; ...                       ...           ...         ...</span>
<span class="co">#&gt; OTU_97.44294 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.45429 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.44375 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.45365 Streptococcaceae Streptococcus F Anaerobic</span>
<span class="co">#&gt; OTU_97.45307 Streptococcaceae Streptococcus F Anaerobic</span></code></pre></div>
</div>
<div id="caching" class="section level1">
<h1 class="hasAnchor">
<a href="#caching" class="anchor"></a>Caching</h1>
<p>The datasets are cached so theyâ€™re only downloaded once. The cache and all of the files contained in it can be removed with the <code>removeCache</code> function.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># removeCache()</span></code></pre></div>
</div>
<div id="convert-to-phyloseq" class="section level1">
<h1 class="hasAnchor">
<a href="#convert-to-phyloseq" class="anchor"></a>Convert to phyloseq</h1>
<p>MicrobiomeBenchmarkData includes function for converting a TreeSummarizedExperiment to a Phyloseq object. This function preservers all of the columns from the rowData, including those columns not related to taxonomy ranks.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">ps</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/toPhyloseq.html">toPhyloseq</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span>
<span class="va">ps</span>
<span class="co">#&gt; phyloseq-class experiment-level object</span>
<span class="co">#&gt; otu_table()   OTU Table:         [ 892 taxa and 76 samples ]</span>
<span class="co">#&gt; sample_data() Sample Data:       [ 76 samples by 8 sample variables ]</span>
<span class="co">#&gt; tax_table()   Taxonomy Table:    [ 892 taxa by 7 taxonomic ranks ]</span>
<span class="co">#&gt; phy_tree()    Phylogenetic Tree: [ 892 tips and 887 internal nodes ]</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="fu">phyloseq</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/phyloseq/man/tax_table-methods.html">tax_table</a></span><span class="op">(</span><span class="va">ps</span><span class="op">)</span><span class="op">)</span>
<span class="co">#&gt; Taxonomy Table:     [6 taxa by 7 taxonomic ranks]:</span>
<span class="co">#&gt;              SUPERKINGDOM PHYLUM       CLASS     ORDER            </span>
<span class="co">#&gt; OTU_97.31247 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt; OTU_97.44487 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt; OTU_97.34979 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt; OTU_97.34572 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt; OTU_97.42259 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt; OTU_97.34698 "Bacteria"   "Firmicutes" "Bacilli" "Lactobacillales"</span>
<span class="co">#&gt;              FAMILY             GENUS           BIOSIS       </span>
<span class="co">#&gt; OTU_97.31247 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span>
<span class="co">#&gt; OTU_97.44487 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span>
<span class="co">#&gt; OTU_97.34979 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span>
<span class="co">#&gt; OTU_97.34572 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span>
<span class="co">#&gt; OTU_97.42259 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span>
<span class="co">#&gt; OTU_97.34698 "Streptococcaceae" "Streptococcus" "F Anaerobic"</span></code></pre></div>
</div>
<div id="session-information" class="section level1">
<h1 class="hasAnchor">
<a href="#session-information" class="anchor"></a>Session information</h1>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; R Under development (unstable) (2021-10-28 r81109)</span>
<span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span>
<span class="co">#&gt; Running under: Ubuntu 20.04.3 LTS</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS:   /home/usuario/Apps/R-devel/lib/libRblas.so</span>
<span class="co">#&gt; LAPACK: /home/usuario/Apps/R-devel/lib/libRlapack.so</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span>
<span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span>
<span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span>
<span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span>
<span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span>
<span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span>
<span class="co">#&gt; [8] base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt;  [1] MicrobiomeBenchmarkData_0.99.0 TreeSummarizedExperiment_2.3.0</span>
<span class="co">#&gt;  [3] Biostrings_2.63.0              XVector_0.35.0                </span>
<span class="co">#&gt;  [5] SingleCellExperiment_1.17.1    SummarizedExperiment_1.25.2   </span>
<span class="co">#&gt;  [7] Biobase_2.55.0                 GenomicRanges_1.47.3          </span>
<span class="co">#&gt;  [9] GenomeInfoDb_1.31.1            IRanges_2.29.0                </span>
<span class="co">#&gt; [11] S4Vectors_0.33.1               BiocGenerics_0.41.1           </span>
<span class="co">#&gt; [13] MatrixGenerics_1.7.0           matrixStats_0.61.0            </span>
<span class="co">#&gt; [15] BiocStyle_2.23.0              </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;   [1] colorspace_2.0-2       ellipsis_0.3.2         rprojroot_2.0.2       </span>
<span class="co">#&gt;   [4] fs_1.5.0               bit64_4.0.5            fansi_0.5.0           </span>
<span class="co">#&gt;   [7] codetools_0.2-18       splines_4.2.0          cachem_1.0.6          </span>
<span class="co">#&gt;  [10] knitr_1.36             ade4_1.7-18            jsonlite_1.7.2        </span>
<span class="co">#&gt;  [13] phyloseq_1.39.0        cluster_2.1.2          dbplyr_2.1.1          </span>
<span class="co">#&gt;  [16] BiocManager_1.30.16    compiler_4.2.0         httr_1.4.2            </span>
<span class="co">#&gt;  [19] assertthat_0.2.1       Matrix_1.3-4           fastmap_1.1.0         </span>
<span class="co">#&gt;  [22] lazyeval_0.2.2         htmltools_0.5.2        tools_4.2.0           </span>
<span class="co">#&gt;  [25] igraph_1.2.8           gtable_0.3.0           glue_1.5.0            </span>
<span class="co">#&gt;  [28] GenomeInfoDbData_1.2.7 reshape2_1.4.4         dplyr_1.0.7           </span>
<span class="co">#&gt;  [31] rappdirs_0.3.3         Rcpp_1.0.7             jquerylib_0.1.4       </span>
<span class="co">#&gt;  [34] pkgdown_1.6.1          vctrs_0.3.8            rhdf5filters_1.7.0    </span>
<span class="co">#&gt;  [37] multtest_2.51.0        ape_5.5                nlme_3.1-153          </span>
<span class="co">#&gt;  [40] iterators_1.0.13       xfun_0.28              stringr_1.4.0         </span>
<span class="co">#&gt;  [43] lifecycle_1.0.1        zlibbioc_1.41.0        MASS_7.3-54           </span>
<span class="co">#&gt;  [46] scales_1.1.1           ragg_1.2.0             parallel_4.2.0        </span>
<span class="co">#&gt;  [49] biomformat_1.23.0      rhdf5_2.39.0           yaml_2.2.1            </span>
<span class="co">#&gt;  [52] curl_4.3.2             memoise_2.0.0          ggplot2_3.3.5         </span>
<span class="co">#&gt;  [55] sass_0.4.0             stringi_1.7.5          RSQLite_2.2.8         </span>
<span class="co">#&gt;  [58] desc_1.4.0             foreach_1.5.1          tidytree_0.3.5        </span>
<span class="co">#&gt;  [61] permute_0.9-5          filelock_1.0.2         BiocParallel_1.29.0   </span>
<span class="co">#&gt;  [64] rlang_0.4.12           pkgconfig_2.0.3        systemfonts_1.0.3     </span>
<span class="co">#&gt;  [67] bitops_1.0-7           evaluate_0.14          lattice_0.20-45       </span>
<span class="co">#&gt;  [70] purrr_0.3.4            Rhdf5lib_1.17.0        treeio_1.19.0         </span>
<span class="co">#&gt;  [73] bit_4.0.4              tidyselect_1.1.1       plyr_1.8.6            </span>
<span class="co">#&gt;  [76] magrittr_2.0.1         bookdown_0.24          R6_2.5.1              </span>
<span class="co">#&gt;  [79] generics_0.1.1         DelayedArray_0.21.1    DBI_1.1.1             </span>
<span class="co">#&gt;  [82] mgcv_1.8-38            pillar_1.6.4           withr_2.4.2           </span>
<span class="co">#&gt;  [85] survival_3.2-13        RCurl_1.98-1.5         tibble_3.1.6          </span>
<span class="co">#&gt;  [88] crayon_1.4.2           utf8_1.2.2             BiocFileCache_2.3.0   </span>
<span class="co">#&gt;  [91] rmarkdown_2.11         grid_4.2.0             data.table_1.14.2     </span>
<span class="co">#&gt;  [94] blob_1.2.2             vegan_2.5-7            digest_0.6.28         </span>
<span class="co">#&gt;  [97] tidyr_1.1.4            textshaping_0.3.6      munsell_0.5.0         </span>
<span class="co">#&gt; [100] bslib_0.3.1</span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by Samuel David Gamboa Tuz.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
