<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Recalibration of the Stammler_2016_16S_spikein dataset • MicrobiomeBenchmarkData</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Recalibration of the Stammler_2016_16S_spikein dataset">
<meta property="og:description" content="MicrobiomeBenchmarkData">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MicrobiomeBenchmarkData</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.0.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/MicrobiomeBenchmarkData.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/datasets.html">Datasets in MicrobiomeBenchmarkData</a>
    </li>
    <li>
      <a href="../articles/recalibrare_spikein_data.html">Recalibration of the Stammler_2016_16S_spikein dataset</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Recalibration of the Stammler_2016_16S_spikein
dataset</h1>
                        <h4 data-toc-skip class="author">Samuel D.
Gamboa-Tuz</h4>
                  <a class="author_email" href="mailto:#"></a><a href="mailto:Samuel.Gamboa.Tuz@gmail.com" class="email">Samuel.Gamboa.Tuz@gmail.com</a>
      
                  
      
      <small class="dont-index">Source: <a href="https://github.com/waldronlab/MicrobiomeBenchmarkData/blob/HEAD/vignettes/recalibrare_spikein_data.Rmd" class="external-link"><code>vignettes/recalibrare_spikein_data.Rmd</code></a></small>
      <div class="hidden name"><code>recalibrare_spikein_data.Rmd</code></div>

    </div>

    
    
<p>One of the main characteristics of the Stammler_2016_16S_spikein
dataset is the presence of spike-in bacteria with a known fixed amount
of bacterial cells. These known loads of bacteria can be used to
recalibrate the raw counts of the matrix and obtain recalibrated
absolute counts. In this vignette, we provide an example of how to
recalibrate the counts of the count matrix based on the read counts of
<em>Salinibacter ruber</em>. This procedure is referred to as
Spike-in-based calibration to total microbial load (SCML) in <a href="https://doi.org/10.1186/s40168-016-0175-0" class="external-link">Sammler et al.,
2016</a>.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/waldronlab/MicrobiomeBenchmarkData" class="external-link">MicrobiomeBenchmarkData</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidyr.tidyverse.org" class="external-link">tidyr</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level3">
<h3 id="import-data">Import data<a class="anchor" aria-label="anchor" href="#import-data"></a>
</h3>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">tse</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/getBenchmarkData.html">getBenchmarkData</a></span><span class="op">(</span><span class="st">'Stammler_2016_16S_spikein'</span>, dryrun <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span><span class="op">[[</span><span class="fl">1</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="co">#&gt; Warning: No taxonomy_tree available for Stammler_2016_16S_spikein.</span></span>
<span><span class="co">#&gt; Finished Stammler_2016_16S_spikein.</span></span>
<span><span class="va">counts</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="ids-of-the-spike-in-bacteria">Ids of the spike-in bacteria<a class="anchor" aria-label="anchor" href="#ids-of-the-spike-in-bacteria"></a>
</h3>
<p>Identifiers of the spiked-in bacteria have the suffix ‘XXXX’.</p>
<table class="table">
<thead><tr class="header">
<th>Bacteria</th>
<th>ID</th>
<th>Load</th>
</tr></thead>
<tbody>
<tr class="odd">
<td><em>Salinibacter ruber</em></td>
<td>AF323500XXXX</td>
<td>3.0 x 10<sup>8</sup>
</td>
</tr>
<tr class="even">
<td><em>Rhizobium radiobacter</em></td>
<td>AB247615XXXX</td>
<td>5.0 x 10<sup>8</sup>
</td>
</tr>
<tr class="odd">
<td><em>Alicyclobacillus acidiphilus</em></td>
<td>AB076660XXXX</td>
<td>1.0 x 10<sup>8</sup>
</td>
</tr>
</tbody>
</table>
</div>
<div class="section level3">
<h3 id="recalibrate-based-on-salinibacter-ruber-abundance-">Recalibrate based on <em>Salinibacter ruber</em> abundance.<a class="anchor" aria-label="anchor" href="#recalibrate-based-on-salinibacter-ruber-abundance-"></a>
</h3>
<p>This recalibration is based on the original article. The only
difference is that the numbers have been rounded up to obtain
counts.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## AF323500XXXX is the unique OTU corresponding to S. ruber</span></span>
<span><span class="va">s_ruber</span> <span class="op">&lt;-</span> <span class="va">counts</span><span class="op">[</span><span class="st">'AF323500XXXX'</span>, <span class="op">]</span></span>
<span><span class="va">size_factor</span> <span class="op">&lt;-</span> <span class="va">s_ruber</span><span class="op">/</span><span class="fu"><a href="https://rdrr.io/r/base/mean.html" class="external-link">mean</a></span><span class="op">(</span><span class="va">s_ruber</span><span class="op">)</span></span>
<span></span>
<span><span class="va">SCML_data</span> <span class="op">&lt;-</span> <span class="va">counts</span> </span>
<span><span class="kw">for</span><span class="op">(</span><span class="va">i</span> <span class="kw">in</span> <span class="fu"><a href="https://rdrr.io/r/base/seq.html" class="external-link">seq</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/nrow.html" class="external-link">ncol</a></span><span class="op">(</span><span class="va">SCML_data</span><span class="op">)</span><span class="op">)</span><span class="op">)</span><span class="op">{</span></span>
<span>    <span class="va">SCML_data</span><span class="op">[</span>,<span class="va">i</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/Round.html" class="external-link">round</a></span><span class="op">(</span><span class="va">SCML_data</span><span class="op">[</span>,<span class="va">i</span><span class="op">]</span> <span class="op">/</span> <span class="va">size_factor</span><span class="op">[</span><span class="va">i</span><span class="op">]</span><span class="op">)</span></span>
<span><span class="op">}</span></span></code></pre></div>
<p>Brief comparison of counts</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="va">no_cal</span> <span class="op">&lt;-</span> <span class="va">counts</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">tibble</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/rownames.html" class="external-link">rownames_to_column</a></span><span class="op">(</span>var <span class="op">=</span> <span class="st">'sample_id'</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">magrittr</span><span class="fu">::</span><span class="fu"><a href="https://magrittr.tidyverse.org/reference/aliases.html" class="external-link">set_colnames</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'sample_id'</span>, <span class="st">'colSum'</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>calibrated <span class="op">=</span> <span class="st">'no'</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html" class="external-link">as_tibble</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">cal</span> <span class="op">&lt;-</span>  <span class="va">SCML_data</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/colSums.html" class="external-link">colSums</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">tibble</span><span class="fu">::</span><span class="fu"><a href="https://tibble.tidyverse.org/reference/rownames.html" class="external-link">rownames_to_column</a></span><span class="op">(</span>var <span class="op">=</span> <span class="st">'sample_id'</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu">magrittr</span><span class="fu">::</span><span class="fu"><a href="https://magrittr.tidyverse.org/reference/aliases.html" class="external-link">set_colnames</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">'sample_id'</span>, <span class="st">'colSum'</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>calibrated <span class="op">=</span> <span class="st">'yes'</span><span class="op">)</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://tibble.tidyverse.org/reference/as_tibble.html" class="external-link">as_tibble</a></span><span class="op">(</span><span class="op">)</span></span>
<span></span>
<span><span class="va">data</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://dplyr.tidyverse.org/reference/bind.html" class="external-link">bind_rows</a></span><span class="op">(</span><span class="va">no_cal</span>, <span class="va">cal</span><span class="op">)</span></span>
<span></span>
<span><span class="va">data</span> <span class="op">|&gt;</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span><span class="va">sample_id</span>, <span class="va">colSum</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> </span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_bar.html" class="external-link">geom_col</a></span><span class="op">(</span><span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>fill <span class="op">=</span> <span class="va">calibrated</span><span class="op">)</span>, position <span class="op">=</span> <span class="st">'dodge'</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggtheme.html" class="external-link">theme_bw</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span></span>
<span>    <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/theme.html" class="external-link">theme</a></span><span class="op">(</span>axis.text.x <span class="op">=</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/element.html" class="external-link">element_text</a></span><span class="op">(</span>angle <span class="op">=</span> <span class="fl">90</span>, hjust <span class="op">=</span> <span class="fl">1</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<p><img src="recalibrare_spikein_data_files/figure-html/unnamed-chunk-4-1.png" width="672"></p>
<p>The counts matrix can be replaced in the original tse in order to
preserve the same metadata.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assay</a></span><span class="op">(</span><span class="va">tse</span><span class="op">)</span> <span class="op">&lt;-</span> <span class="va">SCML_data</span></span>
<span><span class="va">tse</span></span>
<span><span class="co">#&gt; class: TreeSummarizedExperiment </span></span>
<span><span class="co">#&gt; dim: 4036 17 </span></span>
<span><span class="co">#&gt; metadata(0):</span></span>
<span><span class="co">#&gt; assays(1): counts</span></span>
<span><span class="co">#&gt; rownames(4036): GQ448052 EU458484 ... DQ795992 GQ492848</span></span>
<span><span class="co">#&gt; rowData names(1): taxonomy</span></span>
<span><span class="co">#&gt; colnames(17): MID26 MID27 ... MID42 MID43</span></span>
<span><span class="co">#&gt; colData names(12): dataset subject_id ... country description</span></span>
<span><span class="co">#&gt; reducedDimNames(0):</span></span>
<span><span class="co">#&gt; mainExpName: NULL</span></span>
<span><span class="co">#&gt; altExpNames(0):</span></span>
<span><span class="co">#&gt; rowLinks: NULL</span></span>
<span><span class="co">#&gt; rowTree: NULL</span></span>
<span><span class="co">#&gt; colLinks: NULL</span></span>
<span><span class="co">#&gt; colTree: NULL</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; R version 4.2.2 (2022-10-31)</span></span>
<span><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></span>
<span><span class="co">#&gt; Running under: Ubuntu 20.04.5 LTS</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Matrix products: default</span></span>
<span><span class="co">#&gt; BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0</span></span>
<span><span class="co">#&gt; LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; locale:</span></span>
<span><span class="co">#&gt;  [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       </span></span>
<span><span class="co">#&gt;  [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   </span></span>
<span><span class="co">#&gt;  [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          </span></span>
<span><span class="co">#&gt; [10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attached base packages:</span></span>
<span><span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">#&gt; [8] base     </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; other attached packages:</span></span>
<span><span class="co">#&gt;  [1] tidyr_1.2.1                    ggplot2_3.4.0                 </span></span>
<span><span class="co">#&gt;  [3] dplyr_1.0.10                   MicrobiomeBenchmarkData_1.0.0 </span></span>
<span><span class="co">#&gt;  [5] TreeSummarizedExperiment_2.4.0 Biostrings_2.64.1             </span></span>
<span><span class="co">#&gt;  [7] XVector_0.36.0                 SingleCellExperiment_1.18.1   </span></span>
<span><span class="co">#&gt;  [9] SummarizedExperiment_1.26.1    Biobase_2.56.0                </span></span>
<span><span class="co">#&gt; [11] GenomicRanges_1.48.0           GenomeInfoDb_1.32.4           </span></span>
<span><span class="co">#&gt; [13] IRanges_2.30.1                 S4Vectors_0.34.0              </span></span>
<span><span class="co">#&gt; [15] BiocGenerics_0.42.0            MatrixGenerics_1.8.1          </span></span>
<span><span class="co">#&gt; [17] matrixStats_0.62.0             BiocStyle_2.24.0              </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; loaded via a namespace (and not attached):</span></span>
<span><span class="co">#&gt;  [1] nlme_3.1-160           bitops_1.0-7           fs_1.5.2              </span></span>
<span><span class="co">#&gt;  [4] bit64_4.0.5            filelock_1.0.2         httr_1.4.4            </span></span>
<span><span class="co">#&gt;  [7] rprojroot_2.0.3        tools_4.2.2            bslib_0.4.1           </span></span>
<span><span class="co">#&gt; [10] utf8_1.2.2             R6_2.5.1               colorspace_2.0-3      </span></span>
<span><span class="co">#&gt; [13] DBI_1.1.3              lazyeval_0.2.2         withr_2.5.0           </span></span>
<span><span class="co">#&gt; [16] tidyselect_1.2.0       bit_4.0.4              curl_4.3.3            </span></span>
<span><span class="co">#&gt; [19] compiler_4.2.2         textshaping_0.3.6      cli_3.4.1             </span></span>
<span><span class="co">#&gt; [22] desc_1.4.2             DelayedArray_0.22.0    labeling_0.4.2        </span></span>
<span><span class="co">#&gt; [25] bookdown_0.29          sass_0.4.2             scales_1.2.1          </span></span>
<span><span class="co">#&gt; [28] rappdirs_0.3.3         pkgdown_2.0.6          systemfonts_1.0.4     </span></span>
<span><span class="co">#&gt; [31] stringr_1.4.1          digest_0.6.30          yulab.utils_0.0.5     </span></span>
<span><span class="co">#&gt; [34] rmarkdown_2.17         pkgconfig_2.0.3        htmltools_0.5.3       </span></span>
<span><span class="co">#&gt; [37] highr_0.9              dbplyr_2.2.1           fastmap_1.1.0         </span></span>
<span><span class="co">#&gt; [40] rlang_1.0.6            RSQLite_2.2.18         farver_2.1.1          </span></span>
<span><span class="co">#&gt; [43] jquerylib_0.1.4        generics_0.1.3         jsonlite_1.8.3        </span></span>
<span><span class="co">#&gt; [46] BiocParallel_1.30.4    RCurl_1.98-1.9         magrittr_2.0.3        </span></span>
<span><span class="co">#&gt; [49] GenomeInfoDbData_1.2.8 Matrix_1.5-1           munsell_0.5.0         </span></span>
<span><span class="co">#&gt; [52] Rcpp_1.0.9             fansi_1.0.3            ape_5.6-2             </span></span>
<span><span class="co">#&gt; [55] lifecycle_1.0.3        stringi_1.7.8          yaml_2.3.6            </span></span>
<span><span class="co">#&gt; [58] zlibbioc_1.42.0        BiocFileCache_2.4.0    grid_4.2.2            </span></span>
<span><span class="co">#&gt; [61] blob_1.2.3             parallel_4.2.2         crayon_1.5.2          </span></span>
<span><span class="co">#&gt; [64] lattice_0.20-45        knitr_1.40             pillar_1.8.1          </span></span>
<span><span class="co">#&gt; [67] codetools_0.2-18       glue_1.6.2             evaluate_0.18         </span></span>
<span><span class="co">#&gt; [70] BiocManager_1.30.19    vctrs_0.5.0            treeio_1.20.2         </span></span>
<span><span class="co">#&gt; [73] gtable_0.3.1           purrr_0.3.5            assertthat_0.2.1      </span></span>
<span><span class="co">#&gt; [76] cachem_1.0.6           xfun_0.34              tidytree_0.4.1        </span></span>
<span><span class="co">#&gt; [79] ragg_1.2.4             tibble_3.1.8           memoise_2.0.1</span></span></code></pre></div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Samuel Gamboa, Levi Waldron.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
